core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -382,7 +382,8 @@ private[spark] object JsonProtocol {
 
   def executorInfoToJson(executorInfo: ExecutorInfo): JValue = {
     ("Host" -> executorInfo.executorHost) ~
-    ("Total Cores" -> executorInfo.totalCores)
+    ("Total Cores" -> executorInfo.totalCores) ~
+    ("Log Urls" -> mapToJson(executorInfo.logUrlMap))
```

Comment:
Yes this is correct. The previous commit that was blocking this was my patch to add the ExecutorInfo. I totally blanked too so my mistake.